README.md

```diff
@@ -4,7 +4,26 @@ A Kubernetes controller to declaratively manage [Jumperless v5](https://jumperle
 
 ## Description
 
-A Kubernetes controller to declaratively manage [Jumperless v5](https://jumperless-docs.readthedocs.io/)
+**Jumperless v5** is a programmable electronic prototyping device that replaces traditional breadboards with software-controlled connections. Built around an **RP2350B microcontroller**, it enables automated circuit prototyping and testing through:
+
+- **Programmable connections**: Create and modify fully analog electrical connections between nodes using crosspoint matrix switches - no physical jumper wires needed
+- **Arduino Nano compatibility**: Full support for Arduino Nano form factor with UART passthrough and Arduino sketch flashing capabilities
+- **Voltage control**: Set precise voltages (-8V to +8V) on DAC (Digital-to-Analog Converter) channels for power rails and signal generation
+- **Comprehensive I/O**: On-board ADC, current sensing hardware, GPIO, and PWM support for complete circuit control
+- **Development platforms**: Supports both Arduino IDE and MicroPython development environments
```

Comment:
Update this to remove Arduino IDE and update MicroPython reference to indicate the built in interpreter and REPL support. It's not as much about being a developer platform as a way to script, automate, and extend functionality of the device